runtime/runtime/src/verifier.rs

```diff
@@ -295,18 +295,6 @@ pub(crate) fn validate_receipt(
         }
     }
 
-    // We retain these checks here as to maintain backwards compatibility
```

Comment:
we still need those checks until we switch to protocol version 84, so please revert changes to this file